test/tests/functional/pbs_resv_start_dur_end.py

```diff
@@ -48,6 +48,12 @@ class TestReservationRequests(TestFunctional):
     bu = BatchUtils()
     fmt = "%a %b %d %H:%M:%S %Y"
 
+    def setUp(self):
+
+        TestFunctional.setUp(self)
+        self.svrname = self.server.pbs_server_name
+        self.pbs_exec = self.server.pbs_conf['PBS_EXEC']
```

Comment:
I don't see these 2 variables getting use anywhere, what are you using these for?